flocker/dockerplugin/testtools.py

```diff
@@ -0,0 +1,34 @@
+# Copyright ClusterHQ Ltd.  See LICENSE file for details.
+
+"""
+Testing utilities for ``flocker.dockerplugin`.
+"""
+
+from bitmath import TiB, GiB, MiB, KiB
+
+
+def parse_num(expression):
```

Comment:
This needs tests and better error handling. A number of cases that this will fail unintelligibly on:
- Anything that isn't ascii (e.g. `u"\N{snowman}"`)
- Things with non-digits interspersed (e.g. `u"123GiB456")
- Things with multiple runs of non-digits (e.g. `u"glabrazu456GiB"`)
- Things with unknown units (e.g. `u"123parsec"`).
As a general rule, when parsing things, it is better to be explicit in what you accept and reject everything else, rather than trying to strip away what isn't understood and interpreting the rest.

But, probably a better to try and use an existing library (`bitmath.parse_string`?). Unless there is already a specific format that we are parsing (I don't think this is the case).